imports/ui/components/task/task.js

```diff
@@ -4,6 +4,15 @@ import { Tasks } from '../../../api/api.js';
 import './task.html';
 
 Template.task.helpers({
+
+  getStatusColor() {
```

Comment:
Also better practice to put the color hex values in the CSS, and then just apply a class to the html to set the color